wagon.py

```diff
@@ -1191,6 +1302,15 @@ def _add_create_command(parser):
         help='Platform supported by the wagon (eg. linux_x86_64 '
              'or manylinux1).')
 
+    command.add_argument(
+        '--add-file',
+        type=str,
+        nargs='+',
+        default=None,
+        required=False,
+        help='Path to file which would be add to the Wagon.'
```

Comment:
grammar plz, e.g. like this
```suggestion
        help='Path to add to the Wagon.'
```